src/main/java/uk/gov/hmcts/reform/cwrdapi/service/impl/ValidationServiceFacadeImpl.java

```diff
@@ -206,6 +216,38 @@ public void logFailures(String message, long rowId) {
         exceptionCaseWorkerRepository.save(exceptionCaseWorker);
     }
 
+    @Override
+    @Async
+    @Transactional(Transactional.TxType.REQUIRES_NEW)
+    public void saveStaffAudit(AuditStatus auditStatus, String errorMessage,String caseWorkerId,
+                               StaffProfileCreationRequest staffProfileCreationRequest) {
+        try {
+
+            if (errorMessage != null && errorMessage.length() > 512) {
+                errorMessage = errorMessage.substring(0, 511);
+            }
+
+            UserInfo userInfo = jwtGrantedAuthoritiesConverter.getUserInfo();
+            String userId = (nonNull(userInfo) && nonNull(userInfo.getUid())) ? userInfo.getUid() : null;
+            String request = objectMapper.writeValueAsString(staffProfileCreationRequest);
+
+            StaffAudit staffAudit = StaffAudit.builder()
+                        .status(auditStatus.getStatus().toUpperCase())
+                        .requestTimeStamp(LocalDateTime.now())
+                        .errorDescription(errorMessage)
+                        .authenticatedUserId(userId)
+                        .caseWorkerId(caseWorkerId)
+                        .operationType("CREATE")
+                        .requestLog(request)
+                        .build();
+
+            staffAuditRepository.save(staffAudit);
+        } catch (JsonProcessingException e) {
+            log.info("{}:: Failure errorMessager {} in saveStaffAudit {}  ", loggingComponentName, errorMessage,
```

Comment:
I think this should be logged as error instead of info and Can you please correct the typo? It should be "error message" instead of "errorMessager"